semanticdb/semanticdb3/src/main/scala/scala/meta/internal/semanticdb3/Scala.scala

```diff
@@ -14,6 +13,8 @@ object Scala {
     def Global(owner: String, desc: String) = if (owner != RootPackage) owner + desc else desc
     def Local(id: Int): String = Local("local" + id.toString)
     def Local(symbol: String): String = symbol
+    def Multi(symbols: List[String]): String = symbols.mkString(";", ";", "")
+    def parse(symbol: String): (String, Descriptor) = DescriptorParser(symbol).swap
```

Comment:
Looks like this method is only used in tests. If that's the intent, let's move the method into tests.

Reply:
`DescriptorParser.apply` is private, can I make it `private[meta]` instead?

Reply:
Sure.

Reply:
Done.